index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+<header id = "header">
+    <img src = "/images/company-logo.png" alt = "Company Logo" id = "header-img">
+    <nav id = "nav-bar">
+          <button class = "nav-link" href = "#Features">Features</button>
+          <button class = "nav-link" href = "#How_It_Works">How It Works</button>
+          <button class = "nav-link" href = "#Pricing">Pricing</button>
+    </nav>
+   
+   
+    
+</header>
+<div class="div-class">
+  <h1>Prototype Title</h1>
+<video id = "video" controls width = "250">
+<source src = "images/video.mp4" type="video/mp4">
+</video>
+<form id = "form" method="POST" action="https://www.freecodecamp.com/email-submit">
+    <br>
+    <br>
+    <div id = "div-input">
+<input type = "email" id = "email" placeholder="Insert e-mail here..." name = "email">
+<input id="submit" type="submit" value="submit">
+</div>
+<br>
+<br>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    
+    <script  src="js/index.js"></script>
+    <section id = "Features" class = "nav-link">
+    <h2>Features</h2>
+    <h3>Premium Materials</h3>
+    <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase. </p>
+    <h3>Fast Shipping</h3>
+    <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied. </p>
+    <h3>Quality Assurance</h3>
+    <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument. </p>
+    <p></p>
+    </section>
+    <section id = "How_It_Works">
+      <h2>How It Works</h2>
```

Comment:
The comment about the h2 tags are not linked is directed here as well. You can also nest a header element in your section element for the h2 tag!